components/apimgt/org.wso2.carbon.apimgt.core/src/test/resources/testngDAO.xml

```diff
@@ -10,6 +10,7 @@
             <class name="org.wso2.carbon.apimgt.core.dao.impl.LabelDAOImplIT" />
             <class name="org.wso2.carbon.apimgt.core.dao.impl.WorkflowDAOIT" />
             <class name="org.wso2.carbon.apimgt.core.dao.impl.PolicyDAOImplIT" />
+            <class name="org.wso2.carbon.apimgt.core.dao.impl.FunctionDAOImplIT" />
```

Comment:
Mhh I don't think so. Then every test case which involves in dao access become a integration test? having word TestCase will be more convinient. Even our integration tests cases names anded with TestCase